docs/en/stack/ml/limitations.asciidoc

```diff
@@ -208,3 +208,30 @@ pattern that uses the Rollup feature, the {ml} job creation wizards fail.
 
 See {ref}/xpack-rollup.html[Rolling up historical data].
 
+[float]
+[[ml-space-limitations]]
+=== Machine learning objects do not belong to {kib} spaces
+
+If you create {kibana-ref}/xpack-spaces.html[spaces] in {kib}, you see only the  
+saved objects that belong to that space. This limited scope does not apply to 
+{ml} objects; they are visible in all of your spaces.  
+
+However, the {kib} {ml-features} interact with some saved objects (such as 
+index patterns, dashboards, and visualizations) that might not be available in 
+all spaces. For example:
+
+* experimental[] If you upload a file on the *Machine Learning* page in {kib}, 
+the {ml-features} identify the file format and field mappings. You can then 
+optionally import that data into an {es} index and create an index pattern. This 
+index pattern belongs to the space that is active when you import the file. If 
+you want to use the index pattern in other spaces, you must create it as needed.  
+* Likewise, if you clone a {ml} job in a space that does not contain the 
+necessary index pattern, the cloned job fails. 
```

Comment:
To be precise, this only happens if the original job was created with the Single Metric, Multi Metric or Population Wizard. Cloning jobs created via the Advanced Wizard or the API works. And technically, it's not the cloned job that fails, but the (Single Metric, Multi Metric, Population) Wizard fails to load.
Not sure if we want to be that detailed.

Reply:
The changes are looking good @lcawl :+1: